core/src/main/resources/hudson/Messages_de.properties

```diff
@@ -75,3 +75,6 @@ AboutJenkins.DisplayName=Über Jenkins
 AboutJenkins.Description=Versions- und Lizenzinformationen anzeigen.
 
 Functions.NoExceptionDetails=Keine Details zum Ausnahmefehler
+
+DisambiguatedTime.Midnight=0:00 Uhr
+DisambiguatedTime.Noon=12:00 Uhr
```

Comment:
Here you're using the 24h format. Correct would be
```
12:00 Mitternacht
12:00 Mittag
```
But most likely this is unreachable anyway